content/2017/02/22/get-started-designing.rst

```diff
@@ -0,0 +1,104 @@
+public: yes
+author: sondra
+tags: [Tools, Color, Typography, Design, User Experience]
+image:
+  - src: '2017/get-started-designing/cafe-computer.jpg'
+summary: |
+  You'd like to start learning design, but the number of tools and tutorials
+  available is difficult to sort through. Where do you start? I recommend
+  Paper for sketching out UX ideas, *Practice Code Theory for People Who Code*
+  to generate color palettes and learn color theory, *Choosing the Right Font: A Practical Guide to Typography on the Web* as an introduction to typography,
+  and Adobe Experience Design for UX prototyping.
+
+
+Get Started with Design: 4 Useful Tools & Tutorials
+===================================================
+
+So you’re a developer or you’re brand new to the world of web design, and
+you’d like to grow your design chops. How do you even begin to sort through
+the overwhelming amount of tools and tutorials available? In this guide, I’ll
+introduce the tools I use daily in my work as a user experience and brand
+designer for OddBird. I’ll recommend quick and useful tutorials that will get
+you started designing right away.
+
+
+Start Sketching: Paper
+~~~~~~~~~~~~~~~~~~~~~~
+
+When tackling a complex UX design challenge, I start with sketches. My go-to
+tool is aptly named `Paper`_. Paper, by 53, is free for iPad and iPhone. The
+first time I used Paper, I found the interface so clear and simple, I didn’t
+need a tutorial to get started. A collapsable toolkit with several brush
+styles, seven colors, and an eraser is all I need to sketch rough drafts of
+web features. Here’s a sample sketch I drew using Paper during the development
+of QuarqNet.com. With Paper, I can auto-straighten uneven lines and zoom in
+for detail work. When drafting logos, I often trace images. Paper lets me
+upload an image, draw on top of it, and easily delete the image from the
+background when I’m finished.
+
+.. _Paper: https://www.fiftythree.com/
+.. _Practical Color Theory for People Who Code: http://tallys.github.io/color-theory/
+
+
+Color: Sass Color Functions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It’s important for any designer to be able to create pleasing and accessible
+color palettes. If you write code, Natalya Shelburne’s Sass color functions
+are an excellent tool for generating a palette that works. If you’re not a
+developer, Natalya’s `Practical Color Theory for People Who Code`_ is still a
+fantastic tutorial for how to think about color for the web. You’ll learn
+about accessible contrast, using neutral colors, and how to simulate cohesive
+lighting conditions.
+
+Color can make or break a user’s experience on the web. If you’d like to learn
+more, here’s a list of several `fun and easy-to-use tools`_ for generating color
+palettes and testing contrast. I’ve also outlined my step-by-step process for
```

Comment:
Combine "here's a list" intro (kinda vague) with "outlined step by step", so it's more clear what you are linking to up-front?